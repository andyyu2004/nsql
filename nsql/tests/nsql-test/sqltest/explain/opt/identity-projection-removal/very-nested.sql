CREATE TABLE t (id int PRIMARY KEY);
EXPLAIN SELECT * FROM (SELECT * FROM (SELECT * FROM t) JOIN (SELECT * FROM t))
