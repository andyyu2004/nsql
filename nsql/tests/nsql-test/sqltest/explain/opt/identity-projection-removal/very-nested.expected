 cross join    
   scan t (id) 
   scan t (id) 
               